Sources/Plasma/NucleusLib/inc/plPipeline.h

```diff
@@ -357,6 +373,9 @@ class plPipeline : public plCreatable
     
     float fBackingScale = 1.0f;
     void SetBackingScale(float scale) { fBackingScale = scale; };
+    
+    std::shared_ptr<plDisplayHelper> fDisplayHelper;
+    void SetDisplayHelper(std::shared_ptr<plDisplayHelper> helper) { fDisplayHelper = helper; };
```

Comment:
Do we need both the pipeline holding this, and an internal singleton in plDisplayHelper? Is this member variable used anywhere in the pipeline or does it always use `plDisplayHelper::CurrentDisplayHelper()`?

I'm undecided which my preference would be... I guess we need the display helper before we have a pipeline?

Reply:
I think this is leftover from the previous draft. IIRC I didn't want a singleton but it was forced by resolutions needing to be queried during the pipelines constructor.